avm/res/desktop-virtualization/scaling-plan/tests/e2e/max/dependencies.bicep

```diff
@@ -27,7 +30,7 @@ resource hostPool 'Microsoft.DesktopVirtualization/hostPools@2022-09-09' = {
     startVMOnConnect: false
     validationEnvironment: false
     registrationInfo: {
-      expirationTime: '2024-02-24T12:00:00.0000000Z'
+      expirationTime: dateTimeAdd(baseTime, 'P2D')
```

Comment:
I think this may be the wrong place for the function. If we implement it like this, the user will have a hard time to set the exact time they want. Instead we should move the `dateTimeAdd()` to the dafault of the parameters like
```bicep
param baseTime string = dateTimeAdd(utcNow(), 'P2D')
```

@moisesjgomez, thoughts?

Reply:
Sounds reasonable. Something like [this](https://github.com/Azure/bicep-registry-modules/pull/1413/commits/ead442212af76f4bc2d932161d3e7c22cd248f7a)?

Reply:
@krbar @AlexanderSehr I like the idea of moving this to the parameters and within the resource deployment just calling the parameter. In this way, we can default to Current time + 2 days to ensure the pipeline continues to pass tests which is also fine for a deployment as it ensures the deployment would be successful. Then if they want to modify for it to be pushed 30 days out for example, they can modify the 'P2D' to 'P30D' within parameters.

Reply:
@moisesjgomez I already implemented the suggestion of @AlexanderSehr, see: [ead4422](https://github.com/Azure/bicep-registry-modules/pull/1413/commits/ead442212af76f4bc2d932161d3e7c22cd248f7a)

Reply:
@krbar Yes I thought I had approved it but it looks good to me.